### DISCUSSION TITLE
Backup Chatflows

### DISCUSSION THREAD

**Discussion by SHornoiu** [FlowiseAI/Flowise#2134] [Q&A]
Hello,

Long time ago, Flowise had a possibility to EXPORT ALL Chatflows (in the ADMIN menu). After a while, the export can be performed ONLY for EACH Chatflow.

There is any possibility to export ALL Chatflows? How?

Thank you
Sorin

**Comment by toi500** [comment]
No, they removed that option a while ago. They recommend to  back up the database now https://github.com/FlowiseAI/Flowise/issues/1984#issuecomment-2010205834
